install.sh

```diff
@@ -1037,7 +1037,26 @@ download_latest_bundle_and_extract_if_needed() {
     rm -rf $BUILD_DIR
     local URL="https://dl.sandstorm.io/sandstorm-$BUILD.tar.xz"
     echo "Downloading: $URL"
-    curl -A "$CURL_USER_AGENT" -f "$URL" | tar Jxo
+    curl -A "$CURL_USER_AGENT" -f "$URL" > /var/tmp/sandstorm-$BUILD.tar.xz
+    curl -s -A "$CURL_USER_AGENT" -f "$URL.sig" > /var/tmp/sandstorm-$BUILD.tar.xz.sig
+    curl -s -A "$CURL_USER_AGENT" -f "https://dl.sandstorm.io/keyring.gpg" > /var/tmp/sandstorm-keyring.gpg
+
+    if which gpg > /dev/null; then
+      if gpg --no-default-keyring --keyring /var/tmp/sandstorm-keyring.gpg --status-fd 1 \
```

Comment:
Never mind, I stand corrected.

Might still be worth inlining the keyring in the installer script though - one fewer call, and then you also have the keyring signed in your git tag, rather than tying it in later.